packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -34,29 +34,76 @@ type BaseProps = {|
     onClose?: () => void,
 |};
 
-type WrappedProps = {|
-    ...BaseProps,
+export class SmallTwoColumnModal extends React.Component<Props> {
+    render() {
+        const {
+            onClose,
+            sidebar,
+            content,
+            footer,
+            fullBleedSidebar,
+        } = this.props;
 
-    /**
-     * The size of the media layout being used. Populated by MediaLayoutWrapper.
-     * @ignore
-     */
-    mediaSize: MediaSize,
-|};
+        return (
+            <ModalDialog style={styles.smallDialog}>
+                <View style={styles.contentFooterWrapper}>
+                    <View
+                        style={[
+                            styles.contentWrapper,
+                            styles.smallContentWrapper,
+                        ]}
+                    >
+                        <ModalPanel
+                            showCloseButton
+                            color="dark"
+                            onClose={onClose}
+                            style={styles.smallColumn}
+                            content={
+                                fullBleedSidebar ? (
+                                    <ModalContent style={styles.fullBleed}>
+                                        {sidebar}
+                                    </ModalContent>
+                                ) : (
+                                    sidebar
+                                )
+                            }
+                            scrollOverflow={false}
+                        />
+                        <ModalPanel
+                            style={styles.smallColumn}
+                            content={content}
+                            scrollOverflow={false}
+                        />
+                    </View>
+                    {footer && (
```

Comment:
`!!footer`

Reply:
Or as with all the other cases, consider moving to a `maybeRenderFooter` method and then just have that return null when `footer` is falsy.